code/modules/mob/living/carbon/xenomorph/castes/Carrier.dm

```diff
@@ -249,13 +249,19 @@
 		to_chat(src, SPAN_WARNING("You need a facehugger in your hand to throw one!"))
 		return
 
+	if(!do_after(src, 0.5 SECONDS, INTERRUPT_NO_NEEDHAND, BUSY_ICON_HOSTILE, numticks = 1))
+		return
+
 	if(!threw_a_hugger)
 		threw_a_hugger = TRUE
 		for(var/X in actions)
 			var/datum/action/A = X
```

Comment:
Fix single var usage in this proc. See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#dont-use-abbreviations